apps/sms/locales/sms.en-US.properties

```diff
@@ -316,6 +316,39 @@ message-delivery-status-delivered.ariaLabel          = Delivered
 message-delivery-status-read.ariaLabel               = Read
 message-delivery-status-error.ariaLabel              = Error
 
+# Late arrival notices
+late-arrival-notice-in-months         = {[ plural(lateness) ]}
+late-arrival-notice-in-months[zero]   = 
+late-arrival-notice-in-months[one]    = {{ lateness }}mth late
+late-arrival-notice-in-months[other]  = {{ lateness }}mths late
+late-arrival-notice-in-days           = {{ lateness }}d late
+late-arrival-notice-in-hours          = {{ lateness }}h late
+late-arrival-notice-in-minutes        = {{ lateness }}m late
+
+# Lateness message
+message-lateness                = Lateness:
+
+message-lateness-months         = {[ plural(months) ]}
+message-lateness-months[zero]   =
+message-lateness-months[one]    = {{ months }}mth
+message-lateness-months[other]  = {{ months }}mths
+
+message-lateness-days           = {[ plural(days) ]}
+message-lateness-days[zero]     = 
+message-lateness-days[other]    = {{ days }}d
+
+message-lateness-hours          = {[ plural(hours) ]}
+message-lateness-hours[zero]    =
+message-lateness-hours[other]   = {{ hours }}h
+
+message-lateness-minutes        = {[ plural(minutes) ]}
+message-lateness-minutes[zero]  =
+message-lateness-minutes[other] = {{ minutes }}m
+
+message-lateness-seconds        = {[ plural(seconds) ]}
+message-lateness-seconds[zero]  =
+message-lateness-seconds[other] = {{ seconds }}s
+
```

Comment:
Funny you're asking. We just got a first draft of the spec: https://github.com/tc39/ecma402/issues/35 :)

I will be working on shimming it into `moz_intl.js` soon. For our uses I will want to offer async version (using `formatValue`) and DOM version where you pass en element and we assign `data-l10n-id`.

If you want to move this part to moz_intl.js already, feel free! I'll be happy to collaborate on moz_intl.js :)